src/keystore/ram.py

```diff
@@ -353,10 +353,10 @@ async def show_mnemonic(self):
             await self.unlock()
             await self.show(MnemonicScreen(self.mnemonic))
 
-    async def storage_menu(self):
-        """Manage storage and display of the recovery phrase"""
-        # This class can only show mnemonic, can't save
-        await self.show_mnemonic()
+    # async def storage_menu(self):
```

Comment:
So here you made `storage_menu` optional for the keystore.
In this case it makes sense to check if keystore has `storage_menu` attribute and only then display corresponding button.

I think we should do the same with `show_mnemonic` function - potentially `keystore` object may not be able to display a mnemonic (i.e. when all crypto happens on the smartcard and the keys never leave it). Currently it's not the case but better to think about that in advance.

Reply:
Should all be addressed with https://github.com/cryptoadvance/specter-diy/pull/157/commits/bea5a4c9f47dcec1beeb468760a5c99a03c89896